ote/src/cljs/ote/views/service_search.cljs

```diff
@@ -225,7 +224,8 @@
        (doall
          (for [result results]
            ^{:key (::t-service/id result)}
-           [result-card e! (:admin? user) result service-search (:width app)]))
+           [:div
```

Comment:
Did you check if this container div is needed or not? Just a thought.

Reply:
True accidentally left it there when i had both new and old card styles visible. Fixed